src/components/structures/RoomView.tsx

```diff
@@ -423,21 +423,21 @@ export class RoomView extends React.Component<IRoomProps, IRoomState> {
 
             const thread = initialEvent?.getThread();
             if (thread && !initialEvent?.isThreadRoot) {
-                showThread(
-                    thread.rootEvent,
-                    initialEvent,
-                    RoomViewStore.isInitialEventHighlighted(),
-                );
+                showThread({
+                    rootEvent: thread.rootEvent,
+                    initialEvent: initialEvent,
```

Comment:
When the key and value are the same name, you don't need to write both:

```suggestion
                    initialEvent,
```